lxd/seccomp.go

```diff
@@ -871,9 +879,11 @@ func (s *SeccompServer) doDeviceSyscall(c container, args *MknodArgs, siov *Secc
 }
 
 func (s *SeccompServer) HandleMknodSyscall(c container, siov *SeccompIovec) int {
+	logger.Debugf("Handling mknod syscall")
```

Comment:
Can this and following similar Debugf statements be extended to set relevant log.Ctx?
I'd at least expect c.Name(), c.Project() and any relevant information we have about the syscall to be logged.

Because on a busy system, just seeing `Handling mknod syscall` isn't likely to help me much :)

Reply:
Fixed.